example/sinatra/README.md

```diff
@@ -0,0 +1,23 @@
+## Sample Sinatra App
+
+
+### Setup
+
+![](https://camo.githubusercontent.com/35923bd86a1464b04035e6f3b498455f42479244/687474703a2f2f692e696d6775722e636f6d2f5543464c4643632e706e67)
+
+To run this app, you should edit `config.yml` with your Nylas application info and then run the following commands:
```

Comment:
also add here "and make sure you have configured this app's callback URL in your Nylas application". Link to the developer console, also.